content/reference-rippled.md

```diff
@@ -3474,13 +3475,29 @@ The response follows the [standard format](#response-formatting), with a success
 | `ledger_hash`                  | String  | Unique identifying hash of the entire ledger. |
 | `ledger_index`                 | Number  | The [Ledger Index][] of this ledger. |
+| `queue_data`                   | Array   | (Omitted unless requested with the `queue` parameter) Array of objects describing queued transactions, in the same order as the queue. If the request specified `expand` as true, members contain full representations of the transactions, in either JSON or binary depending on whether the request specified `binary` as true. Requires the [FeeEscalation amendment](concept-amendments.html#known-amendments). [New in: rippled 0.70.0][] |
 
 The following fields are deprecated and may be removed without further notice: `accepted`, `hash`, `seqNum`, `totalCoins`.
 
-**Note on `owner_funds`:** If the request specified `"owner_funds": true` and expanded transactions, the response has a field `owner_funds` in the `metaData` object of each [OfferCreate-type transaction](reference-transaction-format.html#offercreate). The purpose of this field is to make it easier to track the [funding status of offers](reference-transaction-format.html#lifecycle-of-an-offer) with each new validated ledger. This field is defined slightly different than the version of this field in [Order Book subscription streams](#order-book-streams):
+Each member of the `queue_data` array represents one transaction in the queue. Some fields of this object may be omitted because they have not yet been calculated. The fields of this object are as follows:
+
+| Field               | Value            | Description                         |
+|:--------------------|:-----------------|:------------------------------------|
+| `account`           | String           | The [Address][] of the sender for this queued transaction. |
+| `tx`                | String or Object | By default, this is a String containing the [identifying hash](#hashes) of the transaction. If transactions are expanded in binary format, this is an object whose only field is `tx_blob`, containing the binary form of the transaction as a decimal string. If transactions are expanded in JSON format, this is an object containing the [transaction object](reference-transaction-format.html) including the transaction's identifying hash in the `hash` field. |
+| `retries_remaining` | Number           | How many times this transaction can be retried before being dropped. |
+| `preflight_result`  | String           | The tentative result from preliminary transaction checking. This is always `tesSUCCESS`. |
+| `last_result`       | String           | _(May be omitted)_ If this transaction was put in the queue after getting a [retriable (`ter`) result](reference-transaction-format.html#result-categories), this is the exact `ter` result code it got. |
```

Comment:
"If this transaction was left in the queue after getting a..."
It's a subtle but important difference. A transaction that gets a retry in the precheck/flight steps will not be put in the queue in the first place. It's only after getting into the queue, then being able to come _out_ because of a retry, will have this set.